README.md

```diff
@@ -0,0 +1,204 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Rentals Frontend](#about-the-project)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage ](#usage-)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Rentals Frontend <a name="about-the-project"></a>
+**Rentals Frontend** is an implemention of a connection between Ruby on Rails back-end and Rentals Frontend. [Rentals Backend](https://github.com/myaserkhan/Rentals-Backend) handles the backend API, and [Rentals Frontend](https://github.com/myaserkhan/Rentals-Frontend) an web app to book an appointment to try a fancy car, to resrve a car.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul> 
+    <li><a href="https://github.com/microverseinc/linters-config/tree/master/ror">Linters</a></li>
+    <li><a href="https://reactjs.org/">React</a></li>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">ROR</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Web app to book an appointment
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- N/A
+
+#### You can login using the following credentials:
+```
+email : foo1@foo.com
+password: admin123
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+### Prerequisites
+
+In order to run this project you need:
+
+- [API](https://github.com/myaserkhan/Rentals-Frontend)
+- [git](https://git-scm.com/)
+- [node.js](https://nodejs.org/en/)
+- [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  git clone git@github.com:myaserkhan/Rentals-Frontend.git
+  cd Rentals-Frontend 
+```
+
+### Install
+
+Install thr project dependencies with:
+
+```
+  npm install
+```
+
+### Usage <a name="usage"></a>
+
+To run the project, execute the following command:
+
+```
+  npm start
+```
+
+### Run tests
+
+To run the test, execute the following command:
+
+```
+  npm test
+```
```

Comment:
- [ ] You are including instructions about how to run tests, however, I couldn't find any in your frontend project.  Kindly either add some frontend tests or remove this line to prevent confusion. 👍